### ISSUE TITLE
[iPad] - timeline should scroll on drag

### ISSUE THREAD

**Issue by kingfisher13** [axismaps/revolt#19]
Since holding the arrow doesn't work (aside from single clicks) due to the ipad's long-press being used by the OS, this would be a reasonable fix.

**Comment by davidheyman** [comment]
Any reason to not make this a fix for all devices? Scrollbars are probably better than the button. What do you think, @bcsheesley?

**Comment by bcsheesley** [comment]
I agree and can't think of any real downside. Let's go with it.

On Tue, Apr 12, 2016 at 2:56 PM, David Heyman notifications@github.com
wrote:

> Any reason to not make this a fix for all devices? Scrollbars are probably
> better than the button. What do you think, @bcsheesley
> https://github.com/bcsheesley?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/axismaps/revolt/issues/19#issuecomment-209078079

## 

---

Ben Sheesley
ben@axismaps.com
(254) 722-8014

Axis Maps LLC

## http://www.axismaps.com

**Comment by kingfisher13** [comment]
@davidheyman @bcsheesley So add the scroll bar to the desktop version as well?

Chrome on Windows:
![image](https://cloud.githubusercontent.com/assets/9831929/14496069/9baeb93e-0157-11e6-8c1a-e2415c4cf61f.png)

Should I remove those left/right buttons then? The buttons match the style of the piece far better than the scroll bar.

**Comment by davidheyman** [comment]
Yes. Please remove the buttons. While you’re right that the Windows scrollbars don’t look great, the buttons did a very bad job at letting people know that there was more timeline (and how much more). Final decision is with Ben, though.
